### ISSUE TITLE
Containers can't write to mounted directories

### ISSUE THREAD

**Issue by jandubois** [lima-vm/lima#206]
From https://github.com/lima-vm/lima/pull/203#issuecomment-913046166:

I'm testing using `/tmp/lima` and it is has weird permissions:

```console
$ ls -ld /tmp/lima
drwxr-xr-x  3 jan  wheel  96  4 Sep 14:42 /tmp/lima
$ lima ls -ld /tmp/lima
drwxr-xr-x 1 root root 96 Sep  4 21:42 /tmp/lima
```

I can create files manually inside the instance with `touch` and `mkdir`, but they end up being owned by root:

```console
jan@lima-default:/tmp/lima$ mkdir foo
jan@lima-default:/tmp/lima$ touch bar
jan@lima-default:/tmp/lima$ ls -l
total 8
-rw-r--r-- 1 root root   0 Sep  4 21:51 bar
drwxr-xr-x 1 root root  64 Sep  4 21:51 foo
drwxr-xr-x 1 root root 288 Sep  4 21:49 xxx
```

But when I try to write to the directory from `nerdctl` it fails:

```console
jan@lima-default:/tmp/lima$ nerdctl run -v /tmp/lima:/mnt screen-static-coreos:latest cp -v /home/core/screen-4.8.0/screen /mnt
cp: cannot stat '/mnt/screen': Permission denied
```

**Comment by AkihiroSuda** [comment]
Thanks for reporting.
I can repro weird permissions and the `Permission denied` error with `nerdctl run -v /tmp/lima:/mnt zoobab/screen-static-coreos:latest cp -v /home/core/screen-4.8.0/screen /mnt `,  but `nerdctl run -v /tmp/lima:/mnt alpine cp /bin/sh /mnt` seems to work 🤔 

Lima v0.6.3 on macOS 11.5 Intel.

**Comment by AkihiroSuda** [comment]
The weird `ls -l` result is being fixed in #211 .

Failure of `zoobab/screen-static-coreos:latest` is expected, because the image uses a custom `USER` in its Dockerfile
https://github.com/zoobab/screen-static-coreos/blob/acc4f4ea23a3aad121becfbbb470a8f208e91521/Dockerfile#L6-L12

```console
$ docker run --rm --tmpfs /mnt zoobab/screen-static-coreos:latest sh -c "id -u && touch /mnt/foo"
1000
touch: cannot touch '/mnt/foo': Permission denied
```
